library/snackbar/snackbar.css

```diff
@@ -0,0 +1,90 @@
+.snackbar {
+  visibility: hidden;
+  min-width: 250px;
+  text-align: center;
+  border-radius: 2px;
+  padding: 0.5rem;
+  margin: 0.5rem;
+  position: fixed;
+  z-index: 1;
+
+  bottom: 30px;
+}
+
+.snackbar-center {
+  margin-left: auto;
+  margin-right: auto;
+  left: 0;
+  right: 0;
+  max-width: 250px;
+}
+
+.snackbar-right {
+  right: 0;
+}
+
+.snackbar-default {
+  background-color: var(--primary-color-light);
+  color: #0000;
+}
+
+.snackbar-success {
+  background-color: rgba(0, 128, 0, 0.5);
+  color: white;
+}
+
+.snackbar-danger {
+  background-color: rgba(255, 0, 0, 0.5);
+  color: white;
+}
+
+.show {
+  visibility: visible !important; /* Show the snackbar */
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+  animation: fadein 0.5s, fadeout 0.5s 2.5s;
+}
+
+/* Animations to fade the snackbar in and out */
+@-webkit-keyframes fadein {
+  from {
+    bottom: 0;
+    /* opacity: 0.7; */
```

Comment:
remove comment if not intended for future use

Reply:
Will use in future, for further enhancement.